ui/src/experiments/list/search/SearchExperimentsFilters.js

```diff
@@ -78,14 +76,14 @@ const SearchExperimentFilters = ({ onChange }) => {
 
   return (
     <>
-      <EuiFlyoutHeader hasBorder className="searchPanelFlyoutHeader">
```

Comment:
It just works? 🎉 Just curious, was this somehow fixed in some version of EUI which was after our current version?

Reply:
Yea it just does. I don't know why the original header's margin was stretched by an additional 20px to the right; maybe it's to enable the 'Reset' button to lie further right than could be done normally?
![Screenshot 2022-09-12 at 12 45 59 AM](https://user-images.githubusercontent.com/36802364/189539310-c3423c7b-42e9-4b4c-a55a-bc408ac2d96a.png)

The same effect is now achieved by making the `EuiFlexItem` containing the button not grow, which causes the `EuiFlexGroup` to allocate more space for the 'Filters' header instead:
![Screenshot 2022-09-12 at 12 43 08 AM](https://user-images.githubusercontent.com/36802364/189539163-3bf20fbe-0ceb-4c67-972a-fdf7fccae262.png)